contrib/mod_sftp/kex.c

```diff
@@ -1649,7 +1649,7 @@ static const char *get_kexinit_hostkey_algo_list(pool *p) {
       if (res > 0) {
         register int j;
 
-        for (j = 0; j < res; i++) {
+        for (j = 0; j < res; j++) {
```

Comment:
When writing this code, I copy/pasted some other `i`-based loop -- and forgot to change all of the occurrences of `i` to `j` (so as to not interfere with the `i` usage by the outermost loop).